code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -666,3 +666,25 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 /atom/proc/create_reagents(const/max_vol)
 	reagents = new/datum/reagents(max_vol)
 	reagents.my_atom = src
+
+///Used by grinders////
+/datum/reagents/proc/grind_item(var/obj/item/I, var/process = 1)
```

Comment:
Honestly it's much better if you added a `/obj/item/proc/grind(var/datum/reagents/reagents, var/obj/machinery/reagentgrinder/grinder, var/process = 1)`, it'd allow you to handle all these istypes in a much more OOP way.

Also it makes just no sense to have it on reagents datums.